cime/config/e3sm/machines/config_machines.xml

```diff
@@ -1418,12 +1418,12 @@
       <modules compiler="intel">
         <command name="load">python</command>
         <command name="load">git</command>
-        <command name="load">intel/18.0.1</command>
+        <command name="load">intel/19.0.4</command>
         <command name="unload">pnetcdf/1.9.0</command>
         <command name="unload">mvapich2</command>
         <command name="unload">cmake</command>
         <command name="load">cmake/3.12.1</command>
-        <command name="load">mvapich2/2.2</command>
+        <command name="load">mvapich2/2.3</command>
```

Comment:
Not sure if this makes any difference but we don't need the `unload` lines anymore.

Reply:
also, just with an eye towards the potential for an upstream merge, I changed the cmake load to version 3.14.5.  I'm imagining that if there is a difference here between E3SM-master and SCREAM there will be a conflict that Jim might have to deal with in the future should he do an upstream merge.

Reply:
```
-        <command name="load">intel/18.0.1</command>
-        <command name="unload">pnetcdf/1.9.0</command>
-        <command name="unload">mvapich2</command>
-        <command name="unload">cmake</command>
-        <command name="load">cmake/3.12.1</command>
-        <command name="load">mvapich2/2.2</command>
+        <command name="load">intel/19.0.4</command>
+        <command name="load">mvapich2/2.3</command>
+        <command name="load">cmake/3.14.5</command>
```